Makefile.prow

```diff
@@ -88,3 +92,16 @@ $(GOBIN)/golangci-lint:
 $(GOBIN)/setup-envtest:
 	@echo "Installing envtest tools"
 	go install sigs.k8s.io/controller-runtime/tools/setup-envtest@latest
+
+.PHONY: clone-oadp-operator
+clone-oadp-operator:
+	@echo "Cloning oadp-operator"
+	rm -rf $(E2E_TEST_DIR)
+	git clone --depth 1 --single-branch --branch $(E2E_TEST_BRANCH) https://github.com/openshift/oadp-operator.git $(E2E_TEST_DIR)
+
+# to run just one test, use `make test-e2e GINKGO_ARGS="--ginkgo.focus='MySQL application CSI'"`
+.PHONY: test-e2e
+test-e2e: clone-oadp-operator
+	@echo "Running oadp-operator e2e tests"
+	pushd $(E2E_TEST_DIR) && export VELERO_IMAGE=$(VELERO_IMAGE) && make test-e2e && popd
+
```

Comment:
how about adding `clean-oadp-operator` that could optionally be ran to clean up downloaded test artifacts?

Reply:
just clarifying.. you are talking about rm -Rf $(E2E_TEST_DIR) ?  happy to add that.  If not that, please clarify

Reply:
that's the one.

Reply:
thanks